cmd/source_profile.go

```diff
@@ -230,8 +285,8 @@ func (src SourceProfile) ToLegacyDriver(source string) (string, error) {
 	}
 }
 
-// Source profile is passed as a list of key value pairs on the command line.
-// Following 3 formats are supported as valid source profiles.
+// Source-profile is passed as a list of key value pairs on the command line.
+// Following 3 formats are supported as valid source-profiles.
```

Comment:
(nit) This looks a little odd - how about "Flag source-profile is passed ....... as a valid source-profile"